### DISCUSSION TITLE
PARKSIDE® Besproeiingscomputer Smart Home »PSBZS A1«

### DISCUSSION THREAD

**Discussion by bartplessers** [Koenkk/zigbee2mqtt#7810] [Devices]
Compatible wit Z2M?
Anybody experience with this device?

https://www.lidl-shop.be/nl-BE/PARKSIDE-Besproeiingscomputer-Smart-Home-PSBZS-A1/p100325201

**Comment by RDJ441** [comment]
Not yet. I'm working on it, but it seems to be quite hard. Hope we can achieve this. ☺

**Comment by careyer** [comment]
Yeah!!! Also highly interested in this!!!!! Seems to be a good device .... =D

**Comment by bartplessers** [comment]
Interesting things on #7695
